source/macos/AdaptiveCards/AdaptiveCards/Renderers/ImageRenderer.swift

```diff
@@ -101,30 +101,43 @@ class ImageRenderer: NSObject, BaseCardElementRendererProtocol {
             logError("imageProperties is null")
             return
         }
-        
+        let constraintIds = ["imageHeight", "imageWidth", "imageAspect", "imageAuto"]
```

Comment:
Can we use string enum instead of typing the string value everytime

Reply:
yes.